### PR TITLE
Improve mod box appearance

### LINKED ISSUE #165: Please change the default icon/thumbnail
I know not much of a "problem" but god it is so ugly and makes it look so bad. It is all pixelated and i feel a better looking one would make it 10x better. sorry for putting a non-real issue in issues, Just voicing a concern.

For further clarification:
/static/background-s.jpg

### PR DESCRIPTION
**Pull request by HebaruSan**
## Motivation

The mod box is somewhat ugly after #293. We gained the ability to follow and download mods in the thumbnail view, but it's done with text links floating over the right middle, which doesn't look good.

image

@V1TA5 created a mock-up of how it could look instead:

image

## Changes

The mod box is reworked to look more like that. The download icon has a very close equivalent in the glyphicons font, and the follow button now follows a "favorites"-like idiom with an empty/filled star in the lower left. Both have tooltips indicating their function ("Follow", "Unfollow", "Download"). And the default mod thumbnail is replaced with a rocket flying out of a box on a dark background.

image

Fixes #165.

Also the user profile now groups mods by game, with a little header showing the game name.

### CODE REVIEW

**Comment by HebaruSan** [comment]
Feedback received via chat:

image

image

mockup

(And a bunch of stuff about how starting over with admin-lte will solve all our problems.)

**Comment by HebaruSan** [comment]
image

image

image

image

**Comment by HebaruSan** [comment]
There's an ongoing discussion in the chat about replacing the default thumbnail, which may or may not happen, but I think the rest of the feedback has been addressed, and this PR should be able to move forward.

File: KerbalStuff/blueprints/profile.py [hunk 1]
```diff
@@ -27,10 +28,16 @@ def view_profile(username: str) -> str:
                 abort(403)
     match = FORUM_PROFILE_URL_PATTERN.match(user.forumUsername)
     forum_url_username = match.groups()[1] if match else None
-    mods_created = sorted(user.mods, key=lambda mod: mod.created, reverse=True)
     # Remove unpublished mods from the list if it's not the accessing's user's own profile, or it is and admin.
-    if not current_user or (current_user.id != user.id and not current_user.admin):
-        mods_created = [mod for mod in mods_created if mod.published]
+    show_unpublished = current_user and (current_user.id == user.id or current_user.admin)
+    mods_created = list(map(lambda grp: (grp[0], sorted(grp[1],
```
Comment (DasSkelett) [inline KerbalStuff/blueprints/profile.py:33]: How about a small comment what all those grouping and sorting lambdas do?
```python
    # Group mods from this profile's user by game, and sort by creation date within each group
```

Comment (HebaruSan) [reply KerbalStuff/blueprints/profile.py:34]: Done.

Comment (DasSkelett) [reply KerbalStuff/blueprints/profile.py:34]: Thanks!

**Review by DasSkelett — APPROVED** [review]
Let's go! :steam_locomotive:

**Pull request merged**